DependencyInjection/CraueConfigExtension.php

```diff
@@ -22,7 +22,13 @@ class CraueConfigExtension extends Extension {
 	 */
 	public function load(array $config, ContainerBuilder $container) {
 		$loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-		$loader->load(Kernel::VERSION_ID < 20700 ? 'form_legacy.xml' : 'form.xml');
+
+		if (Kernel::VERSION_ID < 20700) {
```

Comment:
you should feature detect things based on the form features rather than using the kernel version. the HttpKernel and Form components can be at different versions